AA555A0000000000.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "AA555A0000000000",
```

Comment:
This doesn't look like a real EUI.

Reply:
You are right. I took the wrong number. How can I modify the file?

Reply:
I uploaded the correct file.